HELP.md

```diff
@@ -100,8 +100,8 @@ impl Contract {
         // Verify that the migration can only be done by the owner.
         // This is not necessary, if the upgrade is done internally.
         assert_eq!(
```

Comment:
Should we also be switching over the doc `assert!`s to `require!`s? Might be more beneficial to recommend `require` this way so we can have a singular de facto way of making assertions

Reply:
ah, yeah, this might have swapped back from a merge resolution. Let me switch

Reply:
this will be deleted once https://github.com/near/sdk-docs/pull/37 comes in, but possibly worth adding a point about `require` cc: @mikedotexe